src/Polytopes/Polyhedron/properties.jl

```diff
@@ -3,17 +3,30 @@
 ### Iterators
 ###############################################################################
 ###############################################################################
+"""
+    Polyhedra
+
+Dummy type used for specifying the desired output format.
```

Comment:
... of what? At least say something like "... output format of certain functions dealing with polyhedra?